Formula/e/envoy.rb

```diff
@@ -24,13 +24,13 @@ class Envoy < Formula
   depends_on "bazelisk" => :build
   depends_on "cmake" => :build
   depends_on "libtool" => :build
+  depends_on "llvm@18" => :build
```

Comment:
can we add a note (would be better to include upstream issue) in here?

Reply:
Added a note.

Reply:
Where are these failures reported upstream, and how can we know it's okay to upgrade?